scripts/upload-definition.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/sh
+set -e
+
+chmod +x /scripts/*.sh
+
+[ "_${CCD_DEF_URLS}" = "_" ] && echo "No definitions to load from CCD_DEF_URLS. Script terminated." && exit 0
+
+mkdir /definitions && cd /definitions
+
+defs=$(echo "$CCD_DEF_URLS" | tr "," "\n")
+for def in $defs
+do
+    wget -nd "$def"
+    [ $? -ne 0 ] && "Failed to download \"${def}\". Script terminated." && exit 21
+done
+[ -z "$(ls -A /definitions)" ] && echo "No definitions found to download. Script terminated." && exit 22
+
+if [ ${VERBOSE} = "true" ]; then
+  export CURL_OPTS="-v"
+else
+  export CURL_OPTS="--fail --silent"
+fi
+
+if [ "_${IMPORTER_CREDS_MOUNT}" != "_" ]; then
+  IMPORTER_USERNAME=$(cat ${IMPORTER_CREDS_MOUNT}/ccd-as-a-pr-importer-username)
+  IMPORTER_PASSWORD=$(cat ${IMPORTER_CREDS_MOUNT}/ccd-as-a-pr-importer-password)
+fi
+
+[ "_${CREATE_IMPORTER_USER}" = "_true" ] && /scripts/create-importer-user.sh "${IMPORTER_USERNAME}" "${IMPORTER_PASSWORD}" "${IDAM_URI}"
+
+userToken=$(sh ./scripts/idam-authenticate.sh ${IMPORTER_USERNAME} ${IMPORTER_PASSWORD} ${IDAM_URI} ${REDIRECT_URI} ${CLIENT_ID} ${CLIENT_SECRET})
+
+# add ccd role
+/scripts/add-ccd-role.sh "${CCD_ROLE}" "PUBLIC" "${userToken}" "${API_GATEWAY_BASE_URL}"
+
+for definition in /definitions/C*.xlsx # use C to not process excel temp files that start with ~
```

Comment:
probably don't need this C hack when retrieving remotely?

Reply:
This same image should work locally. I haven't made any breaking changes that I can see.